pkg/util/watch_filters_test.go

```diff
@@ -54,7 +56,7 @@ var _ = Describe("Watch Filters", func() {
 		It("returns a correct request for the cluster ControlPlaneMachineSet", func() {
 			co := configv1resourcebuilder.ClusterOperator().WithName(operatorName).Build()
 
-			Expect(clusterOperatorFilter(co)).To(ConsistOf(reconcile.Request{
+			Expect(clusterOperatorFilter(context.Background(), co)).To(ConsistOf(reconcile.Request{
```

Comment:
Global context variable?
```suggestion
			Expect(clusterOperatorFilter(ctx, co)).To(ConsistOf(reconcile.Request{
```

Reply:
👍